spec/requests/tags_spec.rb

```diff
@@ -17,6 +17,21 @@
       get tags_path
       expect(response.body).not_to include("aliastag")
     end
+
+    it "searches tags" do
```

Comment:
In general, we add the [aggregate_failures](https://relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures) metadata on specs with more than one `expect`:

```suggestion
    it "searches tags", :aggregate_failures do
```